### PR TITLE
Cherednichenko

### PR DESCRIPTION
**Pull request by iCherya**
Guys, please check

### CODE REVIEW

**Review by LessGrits — APPROVED** [review]

File: src/js/AuthUI.js [hunk 1]
```diff
@@ -39,19 +60,28 @@ class AuthUI {
 
     response
       .then((res) => {
-        console.log(res);
         if (res.status === 200) {
           localStorage.setItem('AUTH_TOKEN', res.data.jwt);
+          localStorage.setItem('USER_LOGIN', res.data.user.username);
```
Comment (itbasket) [inline src/js/AuthUI.js]: Work with local storage should be via class User

File: src/js/Board.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+      className: `board ${value}`,
+      children: [heading, stats, cardsContainer, addCardButton],
+      ondragover: this.dragoverHandler.bind(this),
+    });
+    nodeElement.setAttribute('data-value', value);
+
+    this.nodeElement = nodeElement;
+    this.cardsCounterElement = cardsCounterElement;
+    this.cardsContainer = cardsContainer;
+    this.addCardButton = addCardButton;
```
Comment (itbasket) [inline src/js/Board.js]: Some unused variables defined here, please check

**Review by itbasket — CHANGES_REQUESTED** [review]
A bug with adding a card just after login should be fixed.
Also here are some UI/UX improvements I suggest:
- add form clear after an action to all forms
- add min height to the app block
- show loader until cards are loaded

File: src/js/AuthUI.js (hunk 1 above)
Comment (iCherya) [reply src/js/AuthUI.js]: Fixed

File: src/js/Board.js (hunk 2 above)
Comment (iCherya) [reply src/js/Board.js]: Fixed

**Review by LessGrits — APPROVED** [review]

**Review by itbasket — APPROVED** [review]
Great job! Now everything works as it should be

**Pull request merged**